src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/issuegenericletter/IssueGenericLetterAboutToSubmitHandlerTest.java

```diff
@@ -0,0 +1,125 @@
+package uk.gov.hmcts.reform.sscs.ccd.presubmit.issuegenericletter;
+
+import static org.mockito.Mockito.when;
+import static org.mockito.MockitoAnnotations.openMocks;
+import static uk.gov.hmcts.reform.sscs.ccd.callback.CallbackType.ABOUT_TO_SUBMIT;
+import static uk.gov.hmcts.reform.sscs.ccd.domain.EventType.ISSUE_GENERIC_LETTER;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import uk.gov.hmcts.reform.sscs.ccd.callback.Callback;
+import uk.gov.hmcts.reform.sscs.ccd.domain.Appeal;
+import uk.gov.hmcts.reform.sscs.ccd.domain.CaseDetails;
+import uk.gov.hmcts.reform.sscs.ccd.domain.CcdValue;
+import uk.gov.hmcts.reform.sscs.ccd.domain.DocumentSelectionDetails;
+import uk.gov.hmcts.reform.sscs.ccd.domain.DynamicList;
+import uk.gov.hmcts.reform.sscs.ccd.domain.DynamicListItem;
+import uk.gov.hmcts.reform.sscs.ccd.domain.OtherPartySelectionDetails;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+
+class IssueGenericLetterAboutToSubmitHandlerTest {
+
+    private IssueGenericLetterAboutToSubmitHandler handler;
+    @Mock
+    private Callback<SscsCaseData> callback;
+
+    @Mock
+    private CaseDetails<SscsCaseData> caseDetails;
+
+    private SscsCaseData caseData;
+
+
+    private static final String USER_AUTHORISATION = "Bearer token";
+
+    private static final String DOCUMENT_NAME_1 = "DocumentName_1";
+    private static final String DOCUMENT_NAME_2 = "DocumentName_2";
+
+    private static final String CASE_ID = "1111111111111111";
+
+    @BeforeEach
+    protected void setUp() {
+        openMocks(this);
```

Comment:
I think you can forgo this in favour of @ExtendWith(MockitoExtension.class) annotation on the class, but I'm not sure